packages/menhir/menhir.20170509/url

```diff
@@ -0,0 +1,2 @@
+archive: "http://gallium.inria.fr/~fpottier/menhir/menhir-20170509.tar.gz"
```

Comment:
This is causing problems for our offline mirrors that we create from opam, though I don't know if it also causes problems for direct use of opam.